blazor/dropdown-list/form-validation.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: Form validtion in Blazor DropDown List Component | Syncfusion
+description: Checkout and learn here all about Form Validation in Syncfusion Blazor DropDown List component and much more.

```

Comment:
Description: Check out and learn about Form Validation in the Syncfusion Blazor DropDown List component and more.